standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -720,6 +729,15 @@ struct CheckConstraintsResponse {
   1: required list<SQLCheckConstraint> checkConstraints
 }
 
+struct AllTableConstraintsRequest {
+  1: required string dbName,
+  2: required string tblName,
+  3: required string catName
```

Comment:
catName can be optional.

Reply:
3 of the 6 constraint has catName required hence I have to make it required. We can open new jira to make a consistent behaviour across all the constraints.